python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -874,6 +874,9 @@ def _check_solid_defs(self):
         solid_to_pipeline = {}
         for pipeline in self._all_pipelines:
             for solid_def in pipeline.all_node_defs + [pipeline.graph]:
+                if isinstance(solid_def, SubselectedGraphDefinition):
+                    break
```

Comment:
comment at a minimum 

may be worth an api replacement for `all_node_defs` thats more explicit . Kinda weird that graph has to get tacked on here anyway 

theoretically the `parent_graph_def` for any subset one should still get collected here